sqlserver/datadog_checks/sqlserver/queries.py

```diff
@@ -21,3 +21,129 @@
         {'name': 'server.target_memory', 'type': 'gauge'},
     ],
 }
+
+QUERY_AO_FAILOVER_CLUSTER = {
+    'name': 'sys.dm_hadr_cluster',
+    'query': """
+        SELECT
+            cluster_name,
+            quorum_type,
+            quorum_type_desc,
+            quorum_state,
+            quorum_state_desc
+        FROM sys.dm_hadr_cluster
+    """.strip(),
+    'columns': [
+        {'name': 'failover_cluster', 'type': 'tag'},
+        {'name': 'ao.quorum_type', 'type': 'gauge'},
+        {'name': 'quorum_type_desc', 'type': 'tag'},
+        {'name': 'ao.quorum_state', 'type': 'gauge'},
+        {'name': 'quorum_state_desc', 'type': 'tag'},
+    ],
+}
+
+QUERY_AO_FAILOVER_CLUSTER_MEMBER = {
+    'name': 'sys.dm_hadr_cluster_members',
+    'query': """
+        SELECT
+            member_name,
+            member_type,
+            member_type_desc,
+            member_state,
+            member_state_desc,
+            number_of_quorum_votes,
+            FC.cluster_name
+        FROM sys.dm_hadr_cluster_members
+        CROSS APPLY (SELECT cluster_name FROM sys.dm_hadr_cluster) AS FC
+    """.strip(),
+    'columns': [
+        {'name': 'member_name', 'type': 'tag'},
+        {'name': 'ao.member.type', 'type': 'gauge'},
+        {'name': 'member_type_desc', 'type': 'tag'},
```

Comment:
For all `_desc` tag names, drop the `_desc`. I think it makes for a more confusing customer experience.

`failover_mode:automatic` reads better than `failover_mode_desc:automatic`

<img width="425" alt="Screen Shot 2022-05-13 at 11 12 17 AM" src="https://user-images.githubusercontent.com/717862/168313794-54d81434-eff8-4510-b8a0-b351cec25b8f.png">